internal/serve/serve.go

```diff
@@ -141,6 +150,7 @@ func initHandlerDeps(cfg Configs) (handlerDeps, error) {
 		SignatureVerifier:         signatureVerifier,
 		SupportedAssets:           cfg.SupportedAssets,
 		AccountSponsorshipService: accountSponsorshipService,
+		AppTracker:                &cfg.AppTracker,
```

Comment:
Here you don't need to pass the reference, because the interface is already a reference.

```suggestion
		AppTracker:                cfg.AppTracker,
```

Reply:
I need it because the receiver is a pointer (take a look at sentry_tracker.go)